frame/fast-unstake/src/lib.rs

```diff
@@ -288,9 +293,10 @@ pub mod pallet {
 		/// Dispatch origin must be signed by the [`Config::ControlOrigin`].
 		#[pallet::call_index(2)]
 		#[pallet::weight(<T as Config>::WeightInfo::control())]
-		pub fn control(origin: OriginFor<T>, unchecked_eras_to_check: EraIndex) -> DispatchResult {
+		pub fn control(origin: OriginFor<T>, eras_to_check: EraIndex) -> DispatchResult {
 			let _ = T::ControlOrigin::ensure_origin(origin)?;
-			ErasToCheckPerBlock::<T>::put(unchecked_eras_to_check);
+			ensure!(eras_to_check <= T::MaxErasToCheckPerBlock::get(), Error::<T>::CallNotAllowed);
```

Comment:
Could still add a check to the `integrity_test` about the relations between `ErasToCheckPerBlock` and 
`MaxErasToCheckPerBlock` :stuck_out_tongue:

Reply:
Excellent.

Reply:
I went ahead and did that, but `integrity_test` is not run in any particular state, we have to wrap it in a fresh new `TestExternalities`, and in there `ErasToCheckPerBlock` is always zero. So this doesn't check all that much, right?